config/locales/views/en.yml

```diff
@@ -559,6 +559,22 @@ en:
         no_sign_ups_yet: No sign-ups yet!
         offers_html: Offers &#8595;
         requests_html: Requests &#8595;
+  challenge_assignments:
+    index:
+      access_denied_user: You aren't allowed to see that user's assignments.
```

Comment:
This works, but we usually put error messages from the controller in `locales/controllers/en.yml`.